crates/sui-core/src/authority.rs

```diff
@@ -1327,6 +1333,34 @@ impl AuthorityState {
         Ok(self.get_indexes()?.get_transactions_to_addr(address)?)
     }
 
+    /// Returns a full handle to the event store, including inserts... so be careful!
```

Comment:
I'm not sure what this comment is warning about - that the store can be written to?

Reply:
Yeah that's it - that the store can be written to